src/app/src/App.tsx

```diff
@@ -34,22 +90,32 @@ function App() {
     </Box>
   ) : (
     <BrowserRouter>
+      {currentUser ? (
+        <NavBarContainer>
+          <NavBar items={currentUser && currentUser.isAdmin ? adminUser : normalUser} />
+        </NavBarContainer>
+      ) : null}
       <Routes>
         <Route
           index
           element={
             currentUser ? (
-              <Mainpage currentUser={currentUser} />
+              <>
+                <Mainpage currentUser={currentUser} />
+              </>
             ) : (
               <Homepage setCurrentUser={setCurrentUser} />
             )
           }
         />
+        <Route path="/profile/:id" element={<Profile currentUser={currentUser} />} />
+        <Route path="/authors/:id/posts/:id" element={<Comments currentUser={currentUser} />} />
```

Comment:
This looks like the backend route. We wanted the frontend route to just be `path="/post/:postId"` where you can see the post and all it's comments.

Reply:
@Sutanshu I think I was unclear with this, sorry. I DON'T want this to look like a backend route. It is the frontend and route and needs to be exactly `/profile/:authorId/post/:postId`